.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -59,10 +59,10 @@ public static class DateTimeDefinitions
 		public static readonly string HalfYearBackRegex = $@"(the\s+)?(H(?<number>[1-2])|({HalfYearTermRegex}))(\s+of|\s*,\s*)?\s+({YearRegex})";
 		public static readonly string HalfYearRelativeRegex = $@"(the\s+)?{HalfYearTermRegex}(\s+of|\s*,\s*)?\s+({RelativeRegex}\s+year)";
 		public static readonly string AllHalfYearRegex = $@"({HalfYearFrontRegex})|({HalfYearBackRegex})|({HalfYearRelativeRegex})";
-		public const string EarlyPrefixRegex = @"?<EarlyPrefix>early|beginning of|start of";
-		public const string MidPrefixRegex = @"?<MidPrefix>mid|middle of";
-		public const string LaterPrefixRegex = @"?<LatePrefix>late|later(\s+in)?|end of";
-		public static readonly string PrefixPeriodRegex = $@"(({EarlyPrefixRegex})|({MidPrefixRegex})|({LaterPrefixRegex}))";
+		public const string EarlyPrefixRegex = @"(?<EarlyPrefix>early|beginning of|start of)";

```

Comment:
Wouldn't it be best to have "(in\s+the\s+day\s+)" here?

Reply:
This regex is also used in some regexes for datetime period.

Reply:
Is there an impact if the "in the day" part is optional here?

Reply:
It will extract unexpected sentences like "early in the day next month".

Reply:
Fixed.